packages/amesos2/src/Amesos2_MatrixAdapter_def.hpp

```diff
@@ -77,6 +77,22 @@ namespace Amesos2 {
 		typename adapter_t::get_crs_spec());
   }
 
+  template < class Matrix >
+  template<typename KV_S, typename KV_GO, typename KV_GS>
+  void
+  MatrixAdapter<Matrix>::getCrs_kokkos_view(KV_S & nzval,
+        KV_GO & colind,
+        KV_GS & rowptr,
+        typename MatrixAdapter<Matrix>::global_size_t& nnz,
+        const Teuchos::Ptr<const Tpetra::Map<local_ordinal_t, global_ordinal_t, node_t> > rowmap,
+        EStorage_Ordering ordering,
+        EDistribution distribution) const
+  {
```

Comment:
If this is the replacement call for `getCrs` for the new kokkos-view code paths, where does the input `row_map` come from? I think this is missing a call to `getDistributionMap` to create the map needed later to redistribute data to the rank 0 process, which is necessary for most of the direct solvers except superlu_dist (I think that solver hits a different code path). The issue won't be noticed running on a single MPI proc, but would show up when testing with more than one MPI process. See the `getCrs` function below for `getDistributionMap` usage.

Reply:
Simplified the PR so just contains the Tacho serial setup currently.